app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -9,7 +9,7 @@ class Api::V2::ActivationKeysController < Api::V2::ApiController
     before_action :find_activation_key, :only => [:show, :update, :destroy, :available_releases, :copy, :product_content,
                                                   :available_host_collections, :add_host_collections, :remove_host_collections,
                                                   :content_override, :add_subscriptions, :remove_subscriptions,
-                                                  :subscriptions]
+                                                  :subscriptions, :content_overrides]
```

Comment:
This is really the only required change (in Katello) to fix the bug, right?

Reply:
See my comment below, thanks!